src/components/topBar/index.js

```diff
@@ -1,10 +1,27 @@
-import React from "react";
+import React from 'react';
 import "./index.css";
 import imgs from './../../assets/imgs/index';
-import Button from "..//ButtonWithRouter"
+import Button from "..//ButtonWithRouter";
+import { withContext } from './../../provider/index';
+
+export default withContext(({ ctx, beta}) => {
 
-export default ({beta}) => {
   const BetaImg = beta ? <img className="beta-icon" src={beta} alt="beta"/> : null
+
+
+ const joe = () => {

```

Comment:
```suggestion
 const Joe = () => {
```
React components should start with Uppercase letters.

Reply:
`joe` is not a react component but a method. This suggestion should be dismissed